public/app/plugins/panel/pmm-update/components/LastCheck/LastCheck.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC } from 'react';
+
+import { Button, IconName } from '@grafana/ui';
+
+import { LastCheckProps } from '../../types';
+
+import * as styles from './LastCheck.styles';
+
+export const LastCheck: FC<LastCheckProps> = ({ lastCheckDate, onCheckForUpdates, disabled = false }) => (
+  <div className={styles.lastCheck}>
+    <p>
+      Last check: <span data-testid="update-last-check">{lastCheckDate}</span>
```

Comment:
Same as with the other component, would it make sense to have messages file here?